modules/messages/client/components/InboxThread.js

```diff
@@ -10,8 +10,14 @@ export default function InboxThread({ user, thread }) {
   const { t } = useTranslation('messages');
   const otherUser = findOtherUser(user, thread);
   const haveReplied = thread.userFrom._id === user._id;
+  const { read } = thread;
+
   return (
-    <li className="list-group-item threadlist-thread">
+    <li
+      className={`list-group-item threadlist-thread ${
+        !read ? 'threadlist-thread-unread' : ''
```

Comment:
We can use `classnames` package here to simplify; we use it elsewhere in codebase too.